Automattic/pocket-casts-ios
# Transcripts add temporary player item to see transcripts

Refs #1848 

This PR adds a temporary tab view on the player to show a transcripts if available.

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Open an episode with Transcripts ( Cautionary Tales podcast episode have translations)
- Play the episode
- Open the Full Screen Player
- Open the tab for Transcripts
- See if you can see the transcript and scroll around.
- Check that when the episode finishes and a new episode is loaded the transcripts updates correctly.

## Checklist
